guest-tools/run_td.sh

```diff
@@ -10,12 +17,14 @@ cleanup() {
     sleep 3
 }
 
+trap "on_exit" EXIT
+
 cleanup
 if [ "$1" = "clean" ]; then
     exit 0
 fi
 
-TD_IMG=${TD_IMG:-${PWD}/ubuntu-23.10-td.qcow2}
+TD_IMG=${TD_IMG:-${PWD}/image/tdx-guest-ubuntu-23.10.qcow2}
```

Comment:
I'd suggest to **not** do this, but rather do something like
```sh
SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

And then get the image based on the SCRIPT_DIR, as otherwise you'll still see a failure depending from where you call the script.

Reply:
I agree that it would even better if it would not matter from where the script is called

Reply:
the script is supposed to be used following the README, but i will do it, thanks